run_sorting_hat_test_data.sample.sh

```diff
@@ -28,8 +30,18 @@ BIOENTITIES_PROPERTIES=$( pwd )/test-data/bioentity_properties
 SORTING_HAT=${SORTING_HAT:-$( pwd )/Snakefile-sorting-hat}
 LOG_HANDLER=${LOG_HANDLER:-$( pwd )/log_handler.py}
 SN_CONDA_PREFIX=${SN_CONDA_PREFIX:-$( pwd )/conda_installs}
-PROFILE_LINE="--profile profilename"
-LSF_CONFIG=${LSF_CONFIG:-$( pwd )/lsf.yaml}
+
+
+if [ "$CLUSTER" = "LSF" ]; then
+        PROFILE_LINE="--profile lsf-profilename"
+elif [ "$CLUSTER" = "SLURM" ]; then
+        PROFILE_LINE="--profile slurm-profilename"
+else
+        echo "'$CLUSTER' must be either LSF or SLURM"
+        exit 1
+fi
```

Comment:
```suggestion
if [ "$CLUSTER" = "LSF" ]; then
        PROFILE_LINE="--profile lsf-profilename"
        slurm="--slurm"
elif [ "$CLUSTER" = "SLURM" ]; then
        PROFILE_LINE="--profile slurm-profilename"
        slurm=""
else
        echo "'$CLUSTER' must be either LSF or SLURM"
        exit 1
fi
```

Reply:
It'll edit separately as your suggestion should have been `SLURM_OR_NONE="--slurm"` for SLURM